bucket/dosbox-staging.json

```diff
@@ -0,0 +1,48 @@
+{
+    "version": "0.78.1",
+    "description": "DOSBox Staging is DOS/x86 emulator focusing on ease of use. Based on DOSBox.",
+    "homepage": "https://dosbox-staging.github.io/",
+    "license": "GPL-2.0-or-later",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/dosbox-staging/dosbox-staging/releases/download/v0.78.1/dosbox-staging-windows-msys2-x86_64-v0.78.1.zip",
+            "hash": "3c2f408125351154a37e93de8a4bd05d0c722bbf53e1f583909e4ca6c3eb9204"
+        },
+        "32bit": {
+            "url": "https://github.com/dosbox-staging/dosbox-staging/releases/download/v0.78.1/dosbox-staging-windows-msys2-i686-v0.78.1.zip",
+            "hash": "8ffd54502f2411f49d7b9f8e74a7a1861f008fcd1192ed4441262f829c05a608"
+        }
+    },
+    "pre_install": "if (!(Test-Path \"$persist_dir\\dosbox-staging.conf\")) { New-Item -ItemType File \"$dir\\dosbox-staging.conf\" | Out-Null }",
+    "bin": "dosbox.exe",
```

Comment:
```suggestion
    "bin": "dosbox_with_debugger.exe",
```

Reply:
See latest commit which adds the debugger.

See also 3. in the doc for reference, think it's better to have both in the Bin section; https://github.com/dosbox-staging/dosbox-staging/blob/main/README

Reply:
Looking at shortcuts for reference, do I need the same double brackets maybe?